src/zorak/cogs/general/general_trans.py

```diff
@@ -1,74 +1,81 @@
+
 import discord.ext
 import googletrans
 from discord.ext import commands
 from googletrans import Translator
 
+
+def is_multi_lang(lang):
+    if isinstance(lang, list):
```

Comment:
not wrong but cleaner:
`return isinstance(lang, list)`

Reply:
100% correct. This is way easier. And technically, using this we dont need the func, but the func increases readability so i'll leave it in.